src/reducers/proxy.js

```diff
@@ -128,7 +128,10 @@ function useColdAccount(state) {
 
   if (account.singleWallet) {
     return true;
-  } else if (state.onboarding.coldWallet.address !== account.address) {
+  } else if (
+    state.onboarding.coldWallet &&
+    state.onboarding.coldWallet.address !== account.address
```

Comment:
really wish javascript had a null conditional operator

Reply:
add all the babel plugins

![image](https://user-images.githubusercontent.com/24902242/55761666-f521f580-5a14-11e9-95c4-0ac5bd750e7c.png)